administrator/components/com_media/src/Controller/AdaptiveImageController.php

```diff
@@ -42,30 +42,28 @@ public function execute($task)
 		{
 			case "setfocus" :
 				$imgPath = $this->input->getString('path');
+				$width   = $this->input->getInt('width');
 				$dataFocus = array (
-					"box-left"			=> $this->input->getInt('box-left'),
-					"box-top"			=> $this->input->getInt('box-top'),
-					"box-width"			=> $this->input->getInt('box-width'),
+					"box-left"		=> $this->input->getInt('box-left'),
+					"box-top"		=> $this->input->getInt('box-top'),
+					"box-width"		=> $this->input->getInt('box-width'),
 					"box-height"		=> $this->input->getInt('box-height')
```

Comment:
Only -1- space after the longest left-value, all other aligned. NEVER use tabs but for indentation of lines.